src/main/java/seedu/address/model/person/Name.java

```diff
@@ -13,10 +13,10 @@ public class Name {
             + "and it should include given name and surname";
 
     /*
-     * The name must only contain alphanumeric characters and spaces.
+     * The name must only contain alphabetical characters and spaces.
      * It must contain at least two words, for given name and surname.
      */
-    public static final String VALIDATION_REGEX = "[\\p{Alnum}]+ [\\p{Alnum} ]*";
+    public static final String VALIDATION_REGEX = "[a-zA-z]+ [a-zA-z][a-zA-z ]*";
```

Comment:
Shouldn't the regex be `[a-zA-Z]` with the last 'Z' as capital?

Reply:
Also @sannie-beep pointed out that this regex prevents double space, I think the observation is slightly wrong. Yes this does prevent double space, but only after the first word. 
![](https://github.com/user-attachments/assets/16a0cbd4-ab93-4d29-9dd1-076ebe474e51)
`"Alice Meryl  Tan"` is an acceptable name despite the double space after "Meryl"

Perhaps a better regex to capture all these requirements would be `([a-zA-Z]+ )+[a-zA-Z]+`, which accepts 1 to unlimited "word " groups, and end with a "word".

Reply:
```suggestion
    public static final String VALIDATION_REGEX = "([a-zA-Z]+ )+[a-zA-Z]+";
```